src/dlgProfilePreferences.cpp

```diff
@@ -48,6 +48,7 @@
 #include <QTableWidget>
 #include <QToolBar>
 #include <QUiLoader>
+#include <../3rdparty/kdtoolbox/singleshot_connect/singleshot_connect.h>
```

Comment:
I don't think this counts as a "System" library as it is pointing to something within our own "source" materials:
```suggestion
#include "../3rdparty/kdtoolbox/singleshot_connect/singleshot_connect.h"
```